docs/source/AdministratorGuide/Tutorials/basicTutoSetup.rst

```diff
@@ -0,0 +1,765 @@
+    [root@dirac-tuto ~]# yum install -y http://diracproject.web.cern.ch/diracproject/rpm/runit-2.1.2-1.el6.x86_64.rpm
+    Loaded plugins: changelog, kernel-module, protectbase, refresh-packagekit, security, tsflags,
+                  : versionlock
+    Setting up Install Process
+    runit-2.1.2-1.el6.x86_64.rpm                                              | 611 kB     00:00
+    Examining /var/tmp/yum-root-uyeABW/runit-2.1.2-1.el6.x86_64.rpm: runit-2.1.2-1.el6.x86_64
+    Marking /var/tmp/yum-root-uyeABW/runit-2.1.2-1.el6.x86_64.rpm to be installed
+    161 packages excluded due to repository protections
+    Resolving Dependencies
+    --> Running transaction check
+    ---> Package runit.x86_64 0:2.1.2-1.el6 will be installed
+    --> Finished Dependency Resolution
+    Beginning Kernel Module Plugin
+    Finished Kernel Module Plugin
+
+    Dependencies Resolved
+
+    =================================================================================================
+    Package        Arch            Version                 Repository                          Size
+    =================================================================================================
+    Installing:
+    runit          x86_64          2.1.2-1.el6             /runit-2.1.2-1.el6.x86_64          1.7 M
+
+    Transaction Summary
+    =================================================================================================
+    Install       1 Package(s)
+
+    Total size: 1.7 M
+    Installed size: 1.7 M
+    Downloading Packages:
+    Running rpm_check_debug
+    Running Transaction Test
+    Transaction Test Succeeded
+    Running Transaction
+      Installing : runit-2.1.2-1.el6.x86_64                                                      1/1
+    runsvdir start/running, process 7257
+      Verifying  : runit-2.1.2-1.el6.x86_64                                                      1/1
+
+    Installed:
+      runit.x86_64 0:2.1.2-1.el6
+
+    Complete!
+
+
+Next, edit the `/etc/init/runsvdir.conf` file to point to the future DIRAC installation as such::
+
+  # for runit - manage /usr/sbin/runsvdir-start
+  start on runlevel [2345]
+  stop on runlevel [^2345]
+  normal exit 0 111
+  respawn
+  exec /opt/dirac/sbin/runsvdir-start
+
+Finally, create the directory `/opt/dirac/sbin`::
+
+  mkdir -p /opt/dirac/sbin
+
+and the file `/opt/dirac/sbin/runsvdir-start` with the following content::
+
+  cd /opt/dirac
+  RUNSVCTRL='/sbin/runsvctrl'
+  chpst -u dirac $RUNSVCTRL d /opt/dirac/startup/*
+  killall runsv svlogd
+  RUNSVDIR='/sbin/runsvdir'
+  exec chpst -u dirac $RUNSVDIR -P /opt/dirac/startup 'log:  DIRAC runsv'
+
+make it executable::
+
+  chmod +x /opt/dirac/sbin/runsvdir-start
+
+
+and restart `runsvdir`::
+
+  restart runsvdir
+
+
+-------------
+Install MySQL
+-------------
+
+First of all, remove the existing (outdated) installation::
+
+   yum remove -y $(rpm -qa | grep -i mysql | paste -sd ' ')
+
+
+Install all the necessary RPMs for MySQL 5.6::
```

Comment:
Why not 5.7?

Reply:
I had an issue with 5.7 and I did not want to waste time on it, since I wanted first this tutorial out such that we can build on it. If you don't mind, we'll leave it like this for now, and when the tutorial is ready, I'll tackle the issue with 5.7

Reply:
I updated the tuto for MySQL 5.7